### PR TITLE
[WFLY-16297] Add testsuite/integration/elytron to the set of ts modules run for WildFly Preview

### PR DESCRIPTION
**Pull request by bstansberry**
https://issues.redhat.com/browse/WFLY-16297

### CODE REVIEW

**Comment by jamezp** [comment]
The WildFly Preview failure is a real failure. It looks like I forgot to push some changes I had locally.

File: testsuite/integration/elytron/pom.xml [hunk 1]
```diff
@@ -903,6 +905,70 @@
...
+                                <id>default-test</id>
+                                <phase>test</phase>
+                                <configuration>
+                                    <systemPropertyVariables>
+                                        <server.jvm.args>-Dmaven.repo.local=${wildfly.transformed.repo.dir}</server.jvm.args>
+                                    </systemPropertyVariables>
+                                    <!-- These include a module to be used which currently uses javax and is not transformed -->
+                                    <excludedGroups>org.jboss.as.test.shared.categories.RequiresTransformedClass</excludedGroups>
+                                </configuration>
+                            </execution>
```
Comment (jamezp) [inline testsuite/integration/elytron/pom.xml:966]: This is the change I forgot to push for some reason. The `UndertowSSLv2HelloTestCase` will pass by adding:
```xml
<execution>
    <id>sslv2-tests</id>
    <phase>test</phase>
    <configuration>
        <systemPropertyVariables>
            <server.jvm.args>-Dmaven.repo.local=${wildfly.transformed.repo.dir} -Djava.security.properties=${security.properties}</server.jvm.args>
        </systemPropertyVariables>
        <!-- These include a module to be used which currently uses javax and is not transformed -->
        <excludedGroups>org.jboss.as.test.shared.categories.RequiresTransformedClass</excludedGroups>
    </configuration>
</execution>
```

Comment (fjuma) [reply testsuite/integration/elytron/pom.xml:966]: Good catch, @jamezp!

**Comment by bstansberry** [comment]
@fjuma @jamezp I put in James's suggestion above and squashed this down.

**Review by fjuma — APPROVED** [review]
Thanks @bstansberry!

**Pull request merged**